src/Native/Regex.js

```diff
@@ -23,8 +24,19 @@ Elm.Native.Regex.make = function(localRuntime) {
 		return new RegExp(re.source, 'gi');
 	}
 	function regex(raw)
-	{
-		return new RegExp(raw, 'g');
+	{	
+		var compiledRegex = null; 
+		
+		try 
+		{
+			compiledRegex = new RegExp(raw, 'g');
```

Comment:
minor nitpick that you could express this more succinctly as 

```js
return Result.Ok(new RegExp(raw, 'g'));
```

Here, and avoid using the temporary variable - the code would have the same behavior since `new RegExp(raw, 'g')` is evaluated first, would throw, and not instantiate a Result.Ok